File: DSA/TEST_CASES/Python_TestCases/Two_sum.py
class Solution:
    def twoSum(self,List, target):
        check = set()
        for i in range(len(List)):
            if List[i] in check:
                return [List.index(target - List[i]), i]
            else:
                check.add(target - List[i])
                
List=[1,2,3,4,5,6]
TS = Solution()
print(TS.twoSum(List,9))

File: DSA/TEST_CASES/GO_Lang_TestCases/Two_sum.go
package main

import "fmt"

func twoSum(nums []int, target int) []int {
	for i, right := range nums {
		for j, left := range nums {
			if right+left == target && i != j {
				return []int{i, j}
			}
		}
	}
	return nil
}

func main() {
	nums := []int{1, 2, 3, 4, 5, 6}
	TS := twoSum(nums, 9)
	fmt.Println(TS)

}
